test/ProxyEpochRecordFilterTest.cpp

```diff
@@ -240,3 +241,259 @@ TEST_F(ProxyEpochRecordFilterTest, parse_name)
         region_hash, calls_per_epoch, startup_count),
         GEOPM_ERROR_INVALID, "Unable to parse parameter startup_count");
 }
+
+TEST_F(ProxyEpochRecordFilterTest, parse_tutorial_2)
+{
+    const std::string tutorial_2_prof_trace = R"(# geopm_version: 1.1.0+dev241g195e2ed
```

Comment:
It would be nice if these huge blocks of text were in a file that was read in at test time.  It would be much easier to maintain that should the trace format ever change again.

Also keeps this source file from getting bloated.